plugin.rb

```diff
@@ -224,17 +224,18 @@ def accepted_answer
         {
           post_number: info[0],
           username: info[1],
+          excerpt: info[2]
         }
       end
     end
 
     def accepted_answer_post_info
       # TODO: we may already have it in the stream ... so bypass query here
-
-      Post.where(id: accepted_answer_post_id, topic_id: object.topic.id)
-          .joins(:user)
-          .pluck('post_number, username')
-          .first
+      post = Post.where(id: accepted_answer_post_id, topic_id: object.topic.id).joins(:user).first
+      excerpt = post.excerpt
```

Comment:
this is risky, what if post is nil for some reason, topic will explode. also why call excerpt and then excerpt a second time next line. 
I would explicitly only select the data I need here, a bit wasteful to be selecting all these uneeded columns.